src/main.py

```diff
@@ -20,59 +20,108 @@
+                    "text": "Here's what has been shipped during the last 7 days."
+                }
+    },
+])
+
 project_no_stories = []
+
 for project_id in project_ids:
+    message_blocks = []
     # retrieve project stories
     project = pivotal.get_project(project_id)
     stories = pivotal.get_stories(project_id, update_since)
     if len(stories) == 0:
         project_no_stories.append(project_id)
         continue
     # get printable strings
-    printable_stories = get_printable_stories(stories, pivotal.get_project_membership(project_id))
-    slack_message = "*<https://www.pivotaltracker.com/n/projects/%d|%s>*\n" % (project['id'], project['name'])
+    printable_stories = get_printable_stories(
+        stories, pivotal.get_project_membership(project_id))
+    message_blocks.append({
+        "type": "section",
+        "text": {
+            "type": "mrkdwn",
+            "text": "*<https://www.pivotaltracker.com/n/projects/%d|%s>* (%d stories accepted)\n" % (
+                project['id'], project['name'], len(printable_stories))
+        }
+    })
+
     # compose slack message
     for ps in printable_stories:
-        slack_message += "%s\n\n" % ps
-    slack_message += "\n\n"
-    # send message to Slack channel if specified; print to stdout otherwise
+        message_blocks.append({
+            "type": "section",
+            "text": {
+                "type": "mrkdwn",
+                "text": "%s\n\n" % ps
+            }
+        })
+
+    message_blocks.append({
+        "type": "divider"
+    })
+
     if len(slack_channel) > 0:
-        send_slack_message(slack_token, slack_channel, slack_message)
-        send_slack_message(slack_token, slack_channel, "#" * 25)
-    else: 
-        print(slack_message)
-        print("#" * 25)
+        send_slack_message_blocks(slack_token, slack_channel, message_blocks)
+    else:
+        print(message_blocks)
 
-for project_id in project_no_stories:
-    project = pivotal.get_project(project_id)
-    slack_message = "*<https://www.pivotaltracker.com/n/projects/%d|%s>*\n" % (project['id'], project['name'])
-    
+if len(project_no_stories) > 0:
+    parts = []
+    for project_id in project_no_stories:
+        project = pivotal.get_project(project_id)
+        parts.append("*<https://www.pivotaltracker.com/n/projects/%d|%s>*" % (
+            project['id'], project['name']))
     if len(slack_channel) > 0:
-        send_slack_message(slack_token, slack_channel, slack_message)
-        send_slack_message(slack_token, slack_channel, "*no stories*")
-        send_slack_message(slack_token, slack_channel, "#" * 25)
-    else: 
-        print(slack_message)
-        print("*no stories*")
-        print("#" * 25)
+        send_slack_message_blocks(slack_token, slack_channel, [
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": "❗️ No stories accepted for " + ", ".join(parts)
```

Comment:
now 
![Schermata 2020-04-28 alle 09 28 22](https://user-images.githubusercontent.com/822471/80459405-a3efe700-8932-11ea-9730-3342b2f44753.png)

```suggestion
                    "text": "❗️ No stories accepted for \n" + ",\n ".join(parts)
```
![Schermata 2020-04-28 alle 09 28 11](https://user-images.githubusercontent.com/822471/80459446-aeaa7c00-8932-11ea-950c-8086d5d60fdc.png)

Reply:
I actually meant to keep them in the same row, comma separated